rcljava_examples/build.gradle

```
@@ -42,3 +43,12 @@ ament {
         ]
     }
 }
+
+task sourcesJar(type: Jar, dependsOn: classes) {
+    classifier = 'sources'
+    from sourceSets.main.allSource
```

It would be nice to install the artifact to the ament install space. After some playing around, I think this will install the source jar alongside the class jar:
```suggestion
    destinationDir = new java.io.File(ament.installSpace + "/share/" + ament.packageManifestName + "/java")
    from sourceSets.main.allSource
```

In the future, we might want to add an option to the `ament` gradle plugin.